opdracht2.js

```diff
@@ -0,0 +1,29 @@
+// opdracht 2a 33
+let tvSold = 0
+for (let i = 0; i <inventory.length; i++) {
+    tvSold += inventory[i].sold
+}
+console.log(tvSold)
+//Opdracht 2b
+const soldtelevisions = document.getElementById("sold")
+soldtelevisions.textContent =`Het aantal verkochte televisies ${tvSold}`
+//Opdracht 2c 9
+let tvBought = 0
+for (let i = 0; i <inventory.length ; i++) {
+    if (inventory[i].sold===inventory[i].originalStock){
+        tvBought += inventory[i].sold
+    }
```

Comment:
Deze klopt niet helemaal, want je moet ook de níet verkochte tv's meetellen
```suggestion
    tvBought += inventory[i].originalStock;
```